src/status_im/ui/screens/chat/components/input.cljs

```diff
@@ -115,11 +130,11 @@
 (defonce chat-input-key (reagent/atom 1))
 
 (re-frame/reg-fx
- :chat.ui/clear-inputs
- (fn []
-   (reset! input-texts {})
-   (reset! mentions-enabled {})
-   (reset! chat-input-key 1)))
+  :chat.ui/clear-inputs
+  (fn []
+    (reset! input-texts {})
+    (reset! mentions-enabled {})
+    (reset! chat-input-key 1)))
```

Comment:
its weird, lint is failing locally. But tests are showing as passed in pr. And on opening details, it is showing result of first build, not current one.
![image](https://user-images.githubusercontent.com/17097240/188290028-ab438285-d8b5-4266-89a2-804d439934df.png)

Reply:
wow, now as I approved the PR, it also giving me option to merge.
![image](https://user-images.githubusercontent.com/17097240/188290284-c73b6b69-7012-41c6-bceb-1e88bf523733.png)

Maybe @jakubgs , can give it a look

Reply:
Good point. That's because we pipe output into `tee`, but we don't use `pipefail` option. I'll fix it tomorrow.

Reply:
There you go:

* https://github.com/status-im/status-mobile/pull/13926